buildutils/build_cffi.py

```diff
@@ -0,0 +1,128 @@
+import json
+import os
+import cffi
+from zmq.utils.constant_names import all_names, no_prefix
```

Comment:
this import won't succeed unless zmq has already been built and installed, will it?